src/game.cpp

```diff
@@ -13138,31 +13156,79 @@ void game::climb_down( const tripoint &examp )
         you.set_activity_level( ACTIVE_EXERCISE );
         weary_mult = 1.0f / you.exertion_adjusted_move_multiplier( ACTIVE_EXERCISE );
 
-        if( has_grapnel ) {
+        if( can_use_ladder ) {
+            if( !query_yn( _( "Use the ladder to climb down?" ) ) ) {
+                return;
+            } else {
+                has_grapnel = false;
+                web_rappel = false;
+            }
+        }
+        else if( has_grapnel ) {
             if( !query_yn( _( "Use your grappling hook to climb down?" ) ) ) {
                 has_grapnel = false;
             } else {
                 web_rappel = false;
             }
         }
 
-        if( !has_grapnel ) {
+        if( !can_use_ladder && !has_grapnel ) {
             const char *query;
+            const char *climb_back_up = "";
```

Comment:
Does it really need to be a pointer char instead of a string?

Reply:
Query was const char*, so I followed that convention.  I agree that std::string is better in this situation.  I'll change it.  Leaving `query` untouched though.